yarn-project/circuit-types/src/interfaces/block_number.ts

```diff
@@ -0,0 +1,2 @@
+/** Helper type for a specific L2 block number or the latest block number */
+export type BlockNumber = number | 'latest';
```

Comment:
Should it be called L2 block number if only for that. Right now, the type naming don't say anything about it being for the L2, so it might as well be for L1 🤷

Reply:
Addressed